### PR TITLE
[CH] Fix tpcds q67

### PR DESCRIPTION
**Pull request by liuneng1994**
## What changes were proposed in this pull request?

Fix tpcds q67 failed on aqe enabled

## How was this patch tested?

unit tests

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

**Comment by liuneng1994** [comment]
Optimizations for partition 1 do not work

**Comment by lgbo-ustc** [comment]
there is a pr about this in #2153

**Comment by liuneng1994** [comment]
> there is a pr about this part in #2153

the pr cann't work on ds q67, maybe we can remove this optimization directly.

File: gluten-core/src/main/scala/io/glutenproject/execution/TakeOrderedAndProjectExecTransformer.scala [hunk 1]
```diff
@@ -82,19 +83,16 @@ case class TakeOrderedAndProjectExecTransformer(limit: Int,
       }
       val transformStageCounter: AtomicInteger =
         ColumnarCollapseTransformStages.transformStageCounter
-      val finalLimitPlan = if (childRDDPartsNum == 1) {
-          limitExecPlan
```
Comment (ulysses-you) [inline gluten-core/src/main/scala/io/glutenproject/execution/TakeOrderedAndProjectExecTransformer.scala:86]: This is interesting, it seems ok to remove a shuffle if the input rdd only has one partition. Why does it not work ?

Comment (liuneng1994) [reply gluten-core/src/main/scala/io/glutenproject/execution/TakeOrderedAndProjectExecTransformer.scala:86]: Lack of knowledge, I don't know why.

**Pull request closed without merging**